### ISSUE TITLE
Plugin with name Standard_Stream not found. Step name: Standard_Stream

### ISSUE THREAD

**Issue by altjx** [aws/amazon-ssm-agent#370]
When trying to interact with an on-prem agent using the SSM agent, I am receiving the error mentioned in the title. I saw this issue was mentioned back in 2018 here: https://github.com/aws/amazon-ssm-agent/issues/122; however, I'm using a fairly recent version of AWS SSM

I'm using version 3.0.854.0 on this agent, which, according to https://github.com/aws/amazon-ssm-agent/releases, was released 17 days ago.

I have verified that this issue occurs when using the AWS SSM client from the management console as well as the local client installed on my laptop.

Any thoughts?

**Comment by VishnuKarthikRavindran** [comment]
Hi @altjx,

According to me, it looks like a different issue than the older one. To confirm and also to debug, could you please share us the logs from the /var/log/amazon/ssm folder? Sending us the complete reproduction steps will also be helpful.

Thanks,
Vishnu Karthik. R

**Comment by altjx** [comment]
Hey @VishnuKarthikRavindran,

Unfortunately I'm not able to access the agent since it's remote and SSM was the only thing I was relying on. I will reach out to our customer to see if they can work with us on troubleshooting this, and I'll post back any updates.

**Comment by YujiaozhAws** [comment]
Thanks for reaching out. 

Could you please provide the following information? 
1. session manager plugin version. 
2. session manager plugin debug log, see reference. 

Reference: https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-configure-logs

**Comment by altjx** [comment]
Hey guys, I ended up having the client reboot their VM and it looks like everything is working fine now. It's weird though because I was able to get into SSM prior to the issue occurring, but then it just stopped randomly.

@YujiaozhAws is the session manager info necessary if the error is also happening in the web browser?

The interesting thing about this scenario here is it seems to be more on the client (and web browser) side of things. If I run a `tail -f /var/log/amazon/ssm/*` and try to connect using the web console, there is absolutely nothing generated in the client logs. No new entries.
